src/ui/license/LicensePage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Grid, Row, Col } from 'patternfly-react';
+import InternalPage from 'ui/internal-page/InternalPage';
+
+const LicensePage = () => (
+  <InternalPage className="LicensePage">
+    <Grid fluid>
+      <Row>
+        <Col xs={12}>
+          <h2 className="LicensePage__title">
+          License
+          </h2>
+          <section>
+            <p className="LicensePage__description">
+              {`
```

Comment:
why not using simply the text instead of wrapping it in parentheses and backticks?